config/bootstrap.php

```diff
@@ -93,9 +93,9 @@
  * for a very very long time, as we don't want
  * to refresh the cache while users are doing requests.
  */
-if (!Configure::read('debug')) {
-    Configure::write('Cache._cake_model_.duration', '+1 years');
-    Configure::write('Cache._cake_core_.duration', '+1 years');
+if (Configure::read('debug')) {
+    Configure::write('Cache._cake_model_.duration', '+2 minutes');
+    Configure::write('Cache._cake_core_.duration', '+2 minutes');
```

Comment:
Depending on the engine, they might not support to large time-frames.
And IMO 1 year is more than fine. A small re-cache for a single entity at this point in time is acceptable as default for the framework config here.
But the debug mode time should indeed be more like <= 1 minute again.
I think 3.x needs a bit more here since the DB caching of schema is quite time intensive afaik.